### PR TITLE
Db replace module icons

### PR DESCRIPTION
**Pull request by chmst**
With #224 chrome well uses the param header_icon for module icons on dashboards. 
This PR adapts the sql and postgres files.

### CODE REVIEW

File: installation/sql/postgresql/joomla.sql [hunk 1]
```diff
@@ -1459,8 +1459,7 @@ INSERT INTO "#__modules" ("id", "asset_id", "title", "note", "content", "orderin
...
-(9, 44, 'Update Checks', '', '', 3, 'icon', 0, NULL, NULL, NULL, 1, 'mod_quickicon', 3, 1, '{"context":"mod_quickicon","show_jupdate":"1","show_eupdate":"1","show_oupdate":"1","show_privacy":"1","layout":"_:default","moduleclass_sfx":"","cache":1,"cache_time":900,"style":"0","module_tag":"div","bootstrap_size":"12","header_tag":"h3","header_class":"fa-sync"}', 1, '*'),
-(10, 45, 'Logged-in Users', '', '', 2, 'cpanel', 0, NULL, NULL, NULL, 1, 'mod_logged', 3, 1, '{"count":"5","name":"1","layout":"_:default","moduleclass_sfx":"","cache":"0","bootstrap_size": "6"}', 1, '*'),
+(9, 44, 'Update Checks', '', '', 3, 'icon', 0, NULL, NULL, NULL, 1, 'mod_quickicon', 3, 1, '{"context":"mod_quickicon","header_icon":"fa fa-sync","show_jupdate":"1","show_eupdate":"1","show_oupdate":"1","show_privacy":"1","layout":"_:default","moduleclass_sfx":"","cache":1,"cache_time":900,"style":"0","module_tag":"div","bootstrap_size":"12","header_tag":"h3","header_class":""}', 1, '*'),(10, 45, 'Logged-in Users', '', '', 2, 'cpanel', 0, NULL, NULL, NULL, 1, 'mod_logged', 3, 1, '{"count":"5","name":"1","layout":"_:default","moduleclass_sfx":"","cache":"0","bootstrap_size": "6"}', 1, '*'),
```
Comment (brianteeman) [inline installation/sql/postgresql/joomla.sql]: Shouldnt (10, 45, 'Logged -in Users begin on a new line?

**Comment by brianteeman** [comment]
You shouldnt change an update sql file. It needs to be a new file. Otherwise things like db-fix wont see the changes

**Review by bembelimen — DISMISSED** [review]

**Comment by HLeithner** [comment]
> You shouldnt change an update sql file. It needs to be a new file. Otherwise things like db-fix wont see the changes

This is not the main repo and we are not in beta so its perfectly fine to modify the sql file.

**Comment by brianteeman** [comment]
Modified update SQL do not work !!

**Comment by HLeithner** [comment]
I know you have to reinstall.

**Comment by brianteeman** [comment]
You know and I know that it wont work but would anyone testing this PR? Did @chmst know that it wouldn't work? If not then they might make the same mistake on the main repo. There should be no difference in how code is submitted and tested in this repo. The previous behaviour of direct commits without tests or any form of audit history or description was bad enough. Let's not make things worse by allowing code that doesn't work to be committed  I didn't write the issue to be a pia I wrote it because I am trying to test the changes - someone has to as no one else is.

**Comment by HLeithner** [comment]
I really appreciate your engagement and yes sql shouldn't be changed if not needed but this pr doesn't change the structure of the database so you can easily test the change by copy paste.

In the end someone should combine all changes for upgrades between 3.x and 4.0, so he/she has 4 files less to consider.

**Comment by brianteeman** [comment]
I stand by my original comments

**Review by Hackwar — APPROVED** [review]

**Pull request merged**